generator/resources/ec2_mac_test_matrix.json

```diff
@@ -1,7 +1,7 @@
 [
       {
         "os": "macOS Big Sur Arm64",
-        "instanceType":"mac1.metal",
+        "instanceType":"mac2.metal",
         "ami": "amzn-ec2-macos-11.*",
```

Comment:
Where are these AMIs from?
How will we keep them patched?

Reply:
From public AMI and Mac always have a scrubbing feature https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/terraform/setup/main.tf#L77-L78